src/metabase/util/ui_logic.clj

```diff
@@ -83,6 +83,18 @@
   [card results]
   (graph-column-index :graph.dimensions card results))
 
+(defn mult-y-axis-rowfn
+  "This is used as the Y-axis column in the UI
+  when we have comboes, which have more than one y axis."
+  [card results]
+  (let [metrics     (some-> card
+                            (get-in [:visualization_settings :graph.metrics]))
+        col-indices (map #(column-name->index % results) metrics)]
+    (when col-indices
+      (fn [row]
+        (vec (for [idx col-indices]
+               (nth row idx)))))))
```

Comment:
I'm a bit worried that viz settings could be stale and `nth` will throw if it goes out of bounds

Reply:
I have a repro for a case of where it's stale now, damn. willfix